code/modules/clothing/head/helmet.dm

```diff
@@ -947,6 +948,18 @@
 		usr.update_action_buttons()
 		return 1
 
+	if(src && choice && !M.incapacitated() && Adjacent(M))
```

Comment:
If this is true then 939 is true and will never run.

Reply:
So i just remove that as a whole?

Reply:
move the "if(choice == "prime laurel")" to where the first one is and have it be like "If( choice == laurel ||  choice ==whatever)"

Reply:
Like that in my new commit?